packages/core/src/behavior.ts

```diff
@@ -313,12 +316,12 @@ export function createServiceBehavior<TContext, TEvent extends EventObject>(
   service: Interpreter<TContext, TEvent>
 ): Behavior<TEvent, State<TContext, TEvent>> {
   const behavior: Behavior<TEvent, State<TContext, TEvent>> = {
-    receive: (actorContext, event) => {
+    receive: (state, event, actorContext) => {
+      if (isSignal(event)) {
```

Comment:
I wonder now - when does `createServiceBehavior` come useful? it is used to convert a living `service` to the actor ref~ but a service should already be compatible with ActorRef so we should be able to avoid rewrapping it to a similar thing.

I know that service is subtype of the Actor/ActorRef interface but that shouldn't matter much - people should be able to just use it in places where that supertype is expected

Reply:
You're right, it might not need to be wrapped. I'll test it.

Reply:
You were right! 🎉